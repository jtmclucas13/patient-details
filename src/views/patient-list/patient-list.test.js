import React from 'react';
import { shallow } from 'enzyme';
import PatientList from './patient-list';

describe('component PatientList', () => {
    test('renders component', () => {
        const wrapper = renderPatientList();

        expect(wrapper).toMatchSnapshot();
    });
});

function renderPatientList(props = {}) {
    return shallow(<PatientList { ...props } />);
}
