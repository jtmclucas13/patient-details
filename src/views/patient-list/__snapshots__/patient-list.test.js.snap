// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component PatientList renders component 1`] = `
<div
  className="patient-list"
>
  <h2>
    Patient Directory
  </h2>
  <DataTable
    columns={
      Array [
        Object {
          "accessor": [Function],
          "displayName": "Name",
          "key": "name",
        },
        Object {
          "accessor": [Function],
          "displayName": "MRN",
          "key": "mrn",
        },
        Object {
          "accessor": [Function],
          "displayName": "Date of Birth",
          "key": "dob",
        },
        Object {
          "accessor": [Function],
          "displayName": "Demographics",
          "key": "demo",
        },
        Object {
          "accessor": [Function],
          "displayName": "Treatment Site",
          "key": "site",
        },
      ]
    }
    onRowClick={[Function]}
  />
</div>
`;
